config/kibana.yml

```diff
@@ -39,9 +39,10 @@
 #elasticsearch.username: "user"
 #elasticsearch.password: "pass"
 
-# Paths to the PEM-format SSL certificate and SSL key files, respectively. These
-# files enable SSL for outgoing requests from the Kibana server to the browser.
-#server.ssl.cert: /path/to/your/server.crt
+# Enables SSL and paths to the PEM-format SSL certificate and SSL key files, respectively.
+# These settings enable SSL for outgoing requests from the Kibana server to the browser.
+#server.ssl.enabled: true
```

Comment:
thoughts on changing this to false?  for the most part these config examples are set as their defaults